feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -51,6 +53,7 @@ export class AddToCartComponent implements OnInit, OnDestroy {
   @Input() showQuantity = true;
   @Input() options: CartItemComponentOptions;
   @Input() pickupStore: string | undefined;
+  @Input() sapUnit: string;
```

Comment:
MEDIUM: can we make this optional '?', so that it might not throw undefined error in any available custom extensions

Reply:
Done removed